dial_test.go

```diff
@@ -73,7 +74,9 @@ func TestDialer_Dial_passedOpts(t *testing.T) {
 	}
 
 	dialer := &mockPassedDialer{}
-	conn, err := tarantool.Connect(addr, tarantool.Opts{
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
+	conn, err := tarantool.Connect(ctx, addr, tarantool.Opts{
```

Comment:
You don't check that the user context is just passed into the dialer without any modification.

Reply:
Updated. Added more tests for dial.